### DISCUSSION TITLE
DNS resolution is making my response times look high

### DISCUSSION THREAD

**Discussion by BBaoVanC** [upptime/upptime#313] [Ideas]
My response times are high for all of my pages that are on separate subdomains. However, one of my pages is on the same subdomain as an earlier one, so its response time is really low (since the DNS response is cached). Maybe the status checker could do an initial request/ping or something to get the domain cached before checking status?

**Comment by AnandChowdhary** [comment]
Hmm, sounds looks a good idea, do you have a proposal on how we would go about this? Maybe we make a first request, and then a second request? But this means we'll end up making 2 requests for each URL/domain rather than one, right?

**Reply by BBaoVanC** [reply]
One idea is to run a program like `dig` or something beforehand. This could also help with debugging if you wanted to see potential issues with your domain. (Although I'm not exactly sure where the dns caching is done, so having a separate step might not work)
